### DISCUSSION TITLE
Ti problem !

### DISCUSSION THREAD

**Discussion by Rana-Phy** [FitSNAP/FitSNAP#206] [Q&A]
Dear Developers,

Recently I have started to fit snap for Ti. My first target is simple; elastic constants. I found linear snap works very well with hcp Ti dataset (omega, alpha, beta). Problem is arising when I am trying to include Ti's fcc and bcc structures. I think number of distorted structures along the directions are sufficient for fitting elastic constants. However it’s is providing large error, sometine  negative values. I also tried with fcc dataset/bcc dataset separately. Vary the cutoff, group weights using soga. 

I am wondering if there is any fundamental issue regarding capturing fcc and bcc while hcp was fine.

I also see force errors in fcc and bcc group are significantly large. Would you kindly provide some information on error matrices.
Weighted error often 100 times smaller than unweighted error with smartweight=1.  In the example folder of the published snap RMSE in forces is too high (e.g. Fe).  Does the unit for forces eV/atom/Angs. or eV/Angs. in error matrices? 

Best regards,
Rana

**Comment by rohskopf** [comment]
I don't see a fundamental problem in simultaneously matching forces and elastic constants in different phases.

The errors are in eV/Angstrom. There are seemingly large errors e.g. for the Fe example (~300 meV/A). However these linear models were optimized to reproduce other objective functions, and note that forces on simple configurations like the MD groups are relatively lower (< ~100 meV/A). 

It would be useful to plot a distribution of the force values you're trying to match in all groups - do some of the groups have much larger forces than others?

When you tried fcc/bcc separately, are those forces hard to match alone?

Do you have a relatively equal number of configurations in all groups (bcc, fcc, and hcp)?
